src/target/riscv/debug_register_printers.h

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
How about calling the files just `reg_printer.h, .c` or `debug_reg_printer.h, .c`?

That is shorter and, IMO, equally clear.

Reply:
Addressed. The corresponding change in `riscv-debug-spec`: https://github.com/riscv/riscv-debug-spec/pull/876